ypspur_ros_bridge_odom_publisher_node:
  ros__parameters:
    left_wheel: "left_wheel_joint"
    right_wheel: "right_wheel_joint"
    frame_id: "odom"
    child_frame_id: "base_footprint"
